adminSiteClient/GdocsPreviewPage.tsx

```diff
@@ -115,13 +118,23 @@ export const GdocsPreviewPage = ({ match, history }: GdocsMatchProps) => {
     useInterval(async () => {
         if (currentGdoc) {
             const latestGdoc = await fetchGdoc(GdocsContentSource.Gdocs)
+            const iframeScrollY = iframeRef.current?.contentWindow?.scrollY
+            const revisionId = currentGdoc.revisionId
             setCurrentGdoc({
                 ...latestGdoc,
                 slug: currentGdoc.slug,
                 published: currentGdoc.published,
                 publishedAt: currentGdoc.publishedAt,
                 publicationContext: currentGdoc.publicationContext,
             })
+            // Should only run this if the iframe has reloaded
+            setTimeout(() => {
+                if (latestGdoc.revisionId != revisionId) {
```

Comment:
I don't see where `revisionId` is updated?

Reply:
It's part of the spread in `latestGdoc`

It's given to us by Google and it only changes if the Google Doc itself changes, and is also used as the `key` on the iframe below, which is what triggers React to re-render it there, too.